installers/linux/archlinux/PKGBUILD

```diff
@@ -14,7 +15,7 @@ depends=(
 optdepends=(
   'kubectl-bin: to manage the cluster'
   'virtualbox'
-  'docker-machine-kvm'
+  'docker-machine-driver-kvm2'
```

Comment:
Awesome, I see you've added an AUR package for this.  Thanks.

We update this installer automatically during our releases.  Do you mind adding a similar folder for the kvm driver, possibly at `installers/linux/archlinux-drivers` (maybe not the most correct, but so we don't have to move much else around)

And then add a similar hack entry for updating the driver?  If you add minikube-bot as a additional maintainer on AUR, it should be able to update it automatically.

https://github.com/kubernetes/minikube/blob/master/hack/jenkins/release_update_installers.sh#L32-L45

Reply:
Working on this now.